src/rtcModule/webrtc.cpp

```diff
@@ -87,15 +89,60 @@ RtcModule::RtcModule(karere::Client& client, IGlobalHandler& handler,
 
 void RtcModule::init()
 {
-    StaticProvider iceServerStatic(mStaticIceSever);
+    StaticProvider iceServerStatic;
+    string jsonCacheTurnServer = getCachedTurnServers();
+    if (jsonCacheTurnServer.size())
+    {
+        iceServerStatic.setServers(jsonCacheTurnServer.c_str());
+    }
+    else
+    {
+        iceServerStatic.setServers(mStaticIceServers);
+    }
+
     setIceServers(iceServerStatic);
     auto wptr = weakHandle();
     mIceServerProvider.fetchServers()
     .then([wptr, this]()
     {
         if (wptr.deleted())
             return;
-        setIceServers(mIceServerProvider);
+
+        int shard = TURNSERVER_SHARD;
+        for (const std::shared_ptr<TurnServerInfo>& serverInfo : mIceServerProvider)
+        {
+            std::string fullUrl = serverInfo->url;
+            if (fullUrl.size())
+            {
+                size_t posInitialColon = fullUrl.find(":") + 1;
```

Comment:
An example of the `url` here, to understand the format, will help :)

Reply:
Fixed